### PR TITLE
bird: adopt bird flake

### PR DESCRIPTION
**Pull request by MarcelCoding**
This reverts commit ca6d4f9.

### CODE REVIEW

File: modules/bird2.nix [hunk 1]
```diff
@@ -38,128 +38,167 @@ in
...
-              accept;
-            };
-          };
-        }
+      inherit (cfg) routerId;
 
-        protocol kernel kernel_6 {
-          scan time 20;
+      templates.bgp.dn42_peer = ''
+        local as ${builtins.toString cfg.as};
```
Comment (SuperSandro2000) [inline modules/bird2.nix:48]: Maybe we should store the as as a string

Comment (MarcelCoding) [reply modules/bird2.nix:48]: Why? Its a 32bit unsigned integer

Comment (SuperSandro2000) [reply modules/bird2.nix:48]: or use apply toString but minor issue

**Comment by SuperSandro2000** [comment]
@MarcelCoding we tested this succesfull now, did we?

**Comment by MarcelCoding** [comment]
Yes, I've been using it on my flake without problems.

**Comment by SuperSandro2000** [comment]
What about adding a regression test with zentralwerk-network?

**Comment by MarcelCoding** [comment]
It's not yet used in zentralwerk.

**Pull request merged**